test_util/marathon.py

```diff
@@ -0,0 +1,243 @@
+import collections
+import copy
+import logging
+import uuid
+from contextlib import contextmanager
+
+import requests
+import retrying
+
+import test_util.helpers
+
+TEST_APP_NAME_FMT = '/integration-test-{}'
+
+
+def get_test_app(custom_port=False):
+    test_uuid = uuid.uuid4().hex
+    app = copy.deepcopy({
+        'id': TEST_APP_NAME_FMT.format(test_uuid),
+        'cpus': 0.1,
+        'mem': 32,
+        'instances': 1,
+        # NOTE: uses '.' rather than `source`, since `source` only exists in bash and this is
```

Comment:
nit: This one's 100% my fault (need more code reviews of my code...)... this doesn't use `.` or `source` anymore....